src/app/CommandHandler.h

```diff
@@ -426,10 +436,15 @@ class CommandHandler : public Messaging::ExchangeDelegate
 
     bool mSentStatusResponse = false;
 
-    State mState       = State::Idle;
-    bool mGroupRequest = false;
+    State mState = State::Idle;
```

Comment:
Please update `State` to be `enum class State : uint8_t`. This will make the State and 3 bools above "run-in" properly relative to packing, and avoid alignment padding loss.